docs/content/comparisons/druid-vs-commericial-solutions.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: doc_page
+---
+
+Druid vs Proprietary Commercial Solutions (Vertica/Redshift)
```

Comment:
IMO one technology per page is best, so, separate pages for Vertica and Redshift.